bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -11,21 +11,30 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-import cppyy
-
 
 def _getter(k, v):
     # helper function to get CmdArg attribute from `RooFit`
     # Parameters:
     # k: key of the kwarg
     # v: value of the kwarg
+
+    # We have to use ROOT here and not cppy.gbl, because the RooFit namespace is pythonized itself.
+    import ROOT
+
+    func = getattr(ROOT.RooFit, k)
+
+    if func.__class__.__name__ == "CPPOverload":
```

Comment:
Note that you can also compare against `libcppyy.CPPOverload`, which is the type itself, instead of comparing by name. So `if isinstance(func, CPPOverload)`